.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+#### Description
+
+<!--  The description should give a general overview of the goal of the PR.
+
+Individual commit message should highlight important changes that people may
+want to know about year later when sorting thorough the code.
```

Comment:
```suggestion
want to know about year later when sorting through the code.
```
?

Reply:
I copied it from pkcs11-provider and butchered it by mistake :)
Will fix